app/models/comment.rb

```diff
@@ -65,9 +67,18 @@ def set_thread_for_replies
     self.thread = self.commentable.thread if self.reply_comment?
   end
 
-  # Save the ultimate parent
-  def set_parent
+  # Save the ultimate parent and reviewed status
+  def set_parent_and_unreviewed
     self.parent = self.reply_comment? ? self.commentable.parent : self.commentable
+    # we only mark comments as unreviewed if moderated commenting is enabled on their parent
+    self.unreviewed = self.parent.respond_to?(:moderated_commenting_enabled?) && 
+                      self.parent.moderated_commenting_enabled? && 
```

Comment:
Use 2 (not 18) spaces for indenting an expression spanning multiple lines.<br>Redundant `self` detected.<br>Trailing whitespace detected.